cristin-biobank/src/main/java/no/unit/nva/biobank/model/nva/AssociatedProject.java

```diff
@@ -0,0 +1,73 @@
+package no.unit.nva.biobank.model.nva;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import no.unit.nva.biobank.model.cristin.CristinAssociatedProject;
+import no.unit.nva.commons.json.JsonSerializable;
+import nva.commons.core.JacocoGenerated;
+
+import java.beans.ConstructorProperties;
+import java.net.URI;
+import java.util.Map;
+import java.util.Objects;
+
+import static no.unit.nva.utils.UriUtils.createNvaProjectId;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AssociatedProject implements JsonSerializable {
+
+    private final String cristinProjectId;
+    private final Map<String,String> title;
+    private final URI url;
+
+    @ConstructorProperties({"cristinProjectId","title","url"})
```

Comment:
Maybe ask another about the use of ConstructorProperties. Im okay with as long as it works, but could need a second opinion. Nice thing about it is the use of immutables in to and from json